src/mip/platform/serial_connection.cpp

```diff
@@ -42,7 +38,7 @@ bool SerialConnection::isConnected()
 bool SerialConnection::connect()
 {
     if (serial_port_is_open(&mPort))
-     return false;
+        return false;
 
```

Comment:
Connect should return true if the port is already open. The postcondition of this function is that the port is open.